draft-ietf-quic-recovery.md

```diff
@@ -118,10 +118,8 @@ Retransmittable Packets:
 
 All transmissions in QUIC are sent with a packet-level header, which indicates
 the encryption level and includes a packet sequence number (referred to below as
-a packet number).  The encryption level indicates the packet number space, as
-described in {{QUIC-TRANSPORT}}.  Packet numbers never repeat within a packet
-number space for the lifetime of a connection.  Packet numbers monotonically
-increase within a space, preventing ambiguity.
+a packet number).  Packet numbers never repeat for the lifetime of a connection.
+Packet numbers monotonically increase within a space, preventing ambiguity.
```

Comment:
But you have no packet number spaces here....  I'm guessing this is a merge artifact.